www/codeigniter/application/libraries/My_pagination.php

```diff
@@ -267,4 +267,25 @@ class My_Pagination extends CI_Pagination {
 	 * @var	string
 	 */
 	protected $data_page_attr = 'data-ci-pagination-page';
+
+	public function get_last_page_start($input, $count_by_keyword)
```

Comment:
@trandinhlam 
không cần tham số.
sau khi load_pagination(), last_page_start sẽ không thay đổi.
sử dụng $this->total_rows và $this->per_page để tính toán